web-app/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+
+@WebServlet("/login")
+public class LoginServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    UserService userService = UserServiceFactory.getUserService();
+    response.setContentType("text/plain");
```

Comment:
Aren't you restricting yourself in what structured data you'll be able to pass along by making this a plain text response? I wonder how easy it is to serialize or deserialize structured data instead, like JSON or protocol buffers.

If it is too hard, or if you're sure that we will not need structured data here, feel free to ignore this comment.

Reply:
I changed it so that it returns JSON with both url and whether it is logged in or not, to make it easier to get login status!

Reply:
Thanks, Jimin